Source/JavaScriptCore/runtime/StructureInlines.h

```diff
@@ -383,7 +383,13 @@ inline void Structure::didReplaceProperty(PropertyOffset offset)
 {
     if (LIKELY(!isWatchingReplacement()))
         return;
-    firePropertyReplacementWatchpointSet(vm(), offset, "Property did get replaced");
+    didReplacePropertySlow(offset);
+}
+
+inline void Structure::didCachePropertyReplacement(VM& vm, PropertyOffset offset)
```

Comment:
How does this differ from didReplacePropertySlow?

Reply:
Not annotated with EXPORT macro so that it can be only called from JSC framework and LTO inlines it.